Lesson-7/style.css

```diff
@@ -0,0 +1,1071 @@
+    color: #383838;
+    }
+    input[type="date"] {
+    font-family: Roboto;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 18px;
+    line-height: 21px;
+    text-align: center;
+    }
+    input[type="search"]
+    {
+    width: 458px;
+    font-family: Roboto;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 18px;
+    line-height: 21px;
+    color: #383838;
+    }
+    .phone_app {
+    display: flex;
+    justify-content: center;
+    margin-top: 120px;
+    }
+    .icon_google {
+    width: 135px;
+    height: 40px;
+    }
+    .icon_app_store {
+    width: 117px;
+    height: 38px;
+    }
+    .headline {
+    font-family: Roboto;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 40px;
+    line-height: 47px;
+    height: 47px;
+    margin-top: 120px;
+    margin-bottom: 112px;
+    display: flex;
+    justify-content: center;
+    }
+    .container_headphones {
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+    align-items: center;
+    width: 296px;
+    height: 176px;
+    }
+    .container_communicate {
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+    align-items: center;
+    width: 296px;
+    height: 178px;
+    }
+    .container_gadgets {
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+    align-items: center;
+    width: 296px;
+    height: 184px;
+    }
+    .container_like {
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+    align-items: center;
+    width: 296px;
+    height: 183px;
+    }
+    .headphones {
+    width: 75px;
+    Height: 92px;
+    margin-bottom: 52px;
+    }
+    .communicate { 
+    width: 83px;
+    Height: 83px;
+    margin-bottom: 56px;
+    }
+    .gadgets {
+    width: 100px;
+    Height: 67px;
+    margin-bottom: 64px;
+    }
+    .like_img {
+    width: 92px;
+    Height: 83px;
+    margin-bottom: 60px;
+    }
+    .text_headphones {
+    height: 28px;
+    left: 77px;
+    top: 148px;
+    font-family: Roboto;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 24px;
+    line-height: 28px;
+    text-align: center;
+    color: #383838;
+    }
+    .text_communicate {
+    width: 296px;
+    height: 28px;
+    top: 148px;
+    font-family: Roboto;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 24px;
+    line-height: 28px;
+    text-align: center;
+    color: #383838;
+    }
+    .text_gadgets {
+    height: 28px;
+    left: 64px;
+    top: 148px;
+    font-family: Roboto;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 24px;
+    line-height: 28px;
+    text-align: center;
+    color: #383838;
+    }
+    .text_like {
+    width: 225px;
+    height: 28px;
+    left: 36px;
+    top: 148px;
+    font-family: Roboto;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 24px;
+    line-height: 28px;
+    text-align: center;
+    color: #383838;
+    }
```

Comment:
you don't need specify height. Also you don't need specify width for each element. Your containers should have the same width style